.github/workflows/ci.yml

```
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["3.0", "2.7", "2.6", "2.5"]
+        ruby: ["3.1", "3.0", "2.7", "2.6", "2.5"]
```

```suggestion
        ruby: ["3.1", "3.0", "2.7", "2.6"]
```

2.5 is EOL which we no longer support.

---

Sure.  Removed.